app/src/main/java/pl/kossa/akainotes/fragments/notes/NotesFragment.kt

```diff
@@ -1,26 +1,31 @@
 package pl.kossa.akainotes.fragments.notes
 
 import android.os.Bundle
-import android.util.Log
 import android.view.View
 import android.widget.Toast
 import androidx.fragment.app.Fragment
 import androidx.navigation.fragment.findNavController
 import androidx.recyclerview.widget.ItemTouchHelper
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
-import com.google.android.material.snackbar.Snackbar
 import kotlinx.android.synthetic.main.fragment_notes.*
 import pl.kossa.akainotes.R
 import pl.kossa.akainotes.SwipeDeleteCallback
 import pl.kossa.akainotes.adapters.NotesRvAdapter
 import pl.kossa.akainotes.data.Note
 import pl.kossa.akainotes.extensions.makeCancelSnackbar
+import pl.kossa.akainotes.fragments.viewNote.NoteViewViewModel
 
 class NotesFragment : Fragment(R.layout.fragment_notes) {
 
+    private val viewModel = NoteViewViewModel
+
     val adapter by lazy {
-        NotesRvAdapter(arrayListOf())
+        NotesRvAdapter(arrayListOf()) { note ->
+            viewModel.setTitle(note.title.toString())
+            viewModel.setContent(note.description.toString())
+            findNavController().navigate(NotesFragmentDirections.goToNoteView())
```

Comment:
Tak się tego nie robi, można przekazać tytuł i opis przez argumenty -> ale to zostaw to będzie ciekawy przykład na przyszłe spotkanie